spec/controllers/idv/usps_controller_spec.rb

```diff
@@ -69,24 +69,34 @@
       end
 
       it 'calls the UspsConfirmationMaker to send another letter and redirects' do
-        pii = { first_name: 'Samuel', last_name: 'Sampson' }
-        pii_cacher = instance_double(Pii::Cacher)
-        allow(pii_cacher).to receive(:fetch).and_return(pii)
-        allow(Pii::Cacher).to receive(:new).and_return(pii_cacher)
+        resend_letter(otp: false)
+      end
 
-        session[:sp] = { issuer: '123abc' }
+      it 'calls UspsConfirmationMaker to send another letter with reveal_usps_code on' do
+        allow(FeatureManagement).to receive(:reveal_usps_code?).and_return(true)
+        resend_letter(otp: true)
+      end
+    end
+  end
 
-        usps_confirmation_maker = instance_double(UspsConfirmationMaker)
-        allow(UspsConfirmationMaker).to receive(:new).
-          with(pii: pii, issuer: '123abc', profile: pending_profile).
-          and_return(usps_confirmation_maker)
+  def resend_letter(otp:)
```

Comment:
Quick nit: Since this has expectations in it, how would you feel about renaming it something like `expect_resend_letter_to_send_letter_and_redirect`?

Also, looks like you could drop the `otp` param if you pulled line 96 out into each example.

Reply:
Renamed the method.  The otp param is needed because 96 is intertwined with the return value of the instance double